template/page-ambassadors.html.jinja

```diff
@@ -413,6 +268,161 @@
     </div>
 </section>
 
+<!-- Current Students -->
+<section class="container-new py-26">
+
+    <!-- Quotes -->
+    <div id="carouselSlidesOnly" class="carousel slide col-lg-10 mx-auto px-0" data-ride="carousel">
+        <div class="carousel-inner">
+          <div class="carousel-item active">
+            <div class="p-0">
+                <div class="mb-4 p-lg-3">
+                    <img src="./assets/img/ambassadors/quote1-small.svg" class="h-100 d-lg-none mb-4">
+                    <img src="./assets/img/ambassadors/quote1.svg" class="h-100 d-none d-lg-block">
```

Comment:
These images need an `alt` attribute with the text of the quote, for accessibility.